dir.props

```diff
@@ -62,10 +62,6 @@
   <Import Project="$(ToolRuntimePath)BuildVersion.targets" Condition="Exists('$(ToolRuntimePath)BuildVersion.targets')" />
 
   <PropertyGroup>
-    <!-- temporarily increase assembly version of corefx impl of desktop inbox contracts to allow folks
-         to move to the latest packages.  Remove once the full stack is on the latest packages
-         https://github.com/dotnet/corefx/issues/7114 -->
-    <AssemblyVersionTransition>true</AssemblyVersionTransition>
     <GenFacadesArgs>-ignoreBuildAndRevisionMismatch</GenFacadesArgs>
```

Comment:
No, we still have mismatched ref and impl for any contract that has an inbox desktop assembly.  For anything inbox in desktop the ref must match the inbox version exactly.  Once all the refs move to 4.1 we can remove that.